core-bundle/src/Resources/contao/classes/DataContainer.php

```
@@ -1612,9 +1612,16 @@ abstract protected function save($varValue);
 	 * @param string $table
 	 *
 	 * @return string
+	 *
+	 * @todo Change the return type to ?string in Contao 5.0
 	 */
 	public static function getDriverForTable(string $table): string
 	{
+		if (!isset($GLOBALS['TL_DCA'][$table]['config']['dataContainer']))
+		{
+			return '';
```

Should we throw an invalid argument exception instead?

---

Risky but of course it would be correct. It might break extensions though.